config/prod/recover-s3-bucket.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.8/templates/S3/synapse-external-bucket.j2"
+stack_name: "recover-s3-bucket"
+stack_tags:
+  Department: "SysBio"
+  Project: "recover"
```

Comment:
should this be `RECOVER`?

Reply:
AWS uses that tag to query and run reports so it might be good for consistency purposes.

Reply:
👍Will change to 'RECOVER'